### ISSUE TITLE
Profile slips using memary

### ISSUE THREAD

**Issue by AlyaGomaa** [stratosphereips/StratosphereLinuxIPS#200]
Created by Alya Gomaa via monday.com integration. 🎉

**Comment by HAPPYCODER505** [comment]
ok i also want to work on this issue .does it means  memray or should i profile the slips.py file

**Comment by AlyaGomaa** [comment]
yes, use memray to profile slips
All modules starting from slips.py
https://github.com/bloomberg/memray

**Comment by HAPPYCODER505** [comment]
I already downloaded it

**Comment by HAPPYCODER505** [comment]
Ok after gain all the reports. What should I do. Should submited it in the pr or should I try to enhance the performance.
I mean  I think it is a pert of idea 3 in gsoc.
 Improving performance. So it Will help me in gsoc code session.

**Comment by AlyaGomaa** [comment]
yes, detect the part of slips that's using the most memory and tell us how we can optimize it in a PR, also port the results of the profiling  in the PR

**Comment by HAPPYCODER505** [comment]
ok can i use other mamory profilers like memory profiler? and  i also make a profiler .can i use mine also

**Comment by HAPPYCODER505** [comment]
hello.  I profiled slips using memray. and genareted 6 types of reports.so should i make a pr
